vsgenstubs4/_vapoursynth.part.pyi

```diff
@@ -391,17 +400,27 @@ class Core:
     num_threads: int
     max_cache_size: int
 
+    def plugins(self) -> typing.Iterator[Plugin]: ...
+    # get_plugins is deprecated
     def get_plugins(self) -> typing.Dict[str, _PluginMeta]: ...
     def list_functions(self) -> str: ...
 
-    def query_video_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int, subsampling_h: int) -> VideoFormat: ...
+    def query_video_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int = 0, subsampling_h: int = 0) -> VideoFormat: ...
+    # register_format is deprecated
+    def register_format(self, color_family: ColorFamily, sample_type: SampleType, bits_per_sample: int, subsampling_w: int, subsampling_h: int) -> VideoFormat: ...
```

Comment:
Won't including deprecated functions make them suggested an so on for completion? I have no idea

Reply:
Yes, adding them favorises the completion